src/app/CreateRecording/CustomRecordingForm.tsx

```diff
@@ -243,6 +296,80 @@ export const CustomRecordingForm = (props) => {
           </SplitItem>
         </Split>
       </FormGroup>
+      <ExpandableSection toggleTextExpanded="Hide advanced options" toggleTextCollapsed="Show advanced options">
+        <Form>
+          <Text component={TextVariants.small}>
+            A value of 0 for maximum age or size means unbounded.
+          </Text>
+          <FormGroup 
+            label="Maximum age" 
+            fieldId="maxAge"
+            helperText="The maximum age of recording data stored in the JFR repository buffer"
+          >
+            <Split hasGutter={true}>
+              <SplitItem isFilled>
+                <TextInput
+                  value={maxAge}
+                  isRequired
+                  type="number"
+                  id="maxAgeDuration"
+                  aria-label="Max age duration"
+                  onChange={handleMaxAgeChange}
+                  min="0"
+                />
+              </SplitItem>
+              <SplitItem>
+                <FormSelect
+                  value={maxAgeUnits}
+                  onChange={handleMaxAgeUnitChange}
+                  aria-label="Max Age units Input"
+                >
+                  <FormSelectOption key="1" value="1" label="Seconds" />
+                  <FormSelectOption key="2" value={60} label="Minutes" />
+                  <FormSelectOption key="3" value={60*60} label="Hours" />
+                </FormSelect>
+              </SplitItem>
+            </Split>
+          </FormGroup>
+          <FormGroup 
+            label="Maximum size" 
+            fieldId="maxSize"
+            helperText="The maximum size of recording data stored in the JFR repository buffer"
+          >
+            <Split hasGutter={true}>
+              <SplitItem isFilled>
+                  <TextInput
+                    value={maxSize}
+                    isRequired
+                    type="number"
+                    id="maxSize"
+                    aria-label="max size value"
+                    onChange={handleMaxSizeChange}
+                    min="0"
+                  />
+                </SplitItem>
+                <SplitItem>
+                  <FormSelect
+                    value={maxSizeUnits}
+                    onChange={handleMaxSizeUnitChange}
+                    aria-label="Max size units input"
+                  > 
+                    <FormSelectOption key="1" value="1" label="B" />
+                    <FormSelectOption key="2" value={1024} label="KB" />
+                    <FormSelectOption key="3" value={1024*1024} label="MB" />
+                  </FormSelect>
+                </SplitItem>
+            </Split>
+          </FormGroup>
+          <FormGroup fieldId="toDisk">
+            <Checkbox 
```

Comment:
I think `toDisk` should be written as "To Disk" and have a snippet of descriptive text as well, like the numeric options above it have.